recipes/pytalises/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pytalises" %}
+{% set version = "0.1.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytalises-{{ version }}.tar.gz
+  sha256: 880432c41ef1b7bff66b48a370cd596f6fdf8b4290cbb18c1e2df873e5f84576
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - llvmlite >=0.34
+    - numba >=0.50
+    - numexpr
+    - numpy
+    - pyfftw
+    - scipy
+  run:
+    - llvmlite >=0.34
+    - numba >=0.50
+    - numexpr
+    - numpy
+    - pyfftw
+    - python >=3.6
+    - scipy
+
+test:
+  imports:
+    - pytalises
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/savowe/pytalises
+  license: GPL-3.0-only
+  license_family: GPL3
```

Comment:
I think it should be `license_family: GPL`

Reply:
GPL3 is a allowed member of license families according to https://github.com/conda/conda-build/blob/master/conda_build/license_family.py
Now I do not know whether it necessary or good practice to differentiate the three versions of GPL as different license families. Shall I change it?